components/data-oriented-programming/code/java-bbe/optional-types.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: 'Model optionality'
-description: In data-oriented programming, where data takes precedence, modeling optionality is crucial for expressing the presence or absence of data in a concise and type-safe manner. <br><br>Optional typing enables the indication of nullable or absent values, while optional fields offer flexibility in representing different data states.<br><br>Ballerina has built-in support for optional types and fields, eliminating the risk of null pointer exceptions and related bugs, ensuring more robust code.
+description: In data-oriented programming where data takes precedence, modeling optionality is crucial for expressing the presence or absence of data in a concise and type-safe manner. <br><br>Optional typing enables the indication of nullable or absent values while optional fields offer flexibility in representing different data states.<br><br>Ballerina has built-in support for optional types and fields, eliminating the risk of null pointer exceptions and related bugs and ensuring more robust code.
```

Comment:
Isn't it correct to have comma here? as it emphasizes `ensuring more robust code` part which we need to achieve.

Reply:
It depends on to which part of the sentence `ensuring more robust code` belongs.

It should be that both `eliminating the risk of null pointer exceptions and related bugs` and `ensuring more robust code` are the results of `Ballerina has built-in support for optional types and fields`. Therefore we need to combine the 2 with the word `and`.

If `ensuring more robust code` is a result of only `eliminating the risk of null pointer exceptions and related bugs` and doesn't apply to the first part of the sentence, then, the comma in the original text is correct.

Reply:
Second case explains what's intended

Reply:
Added the comma back.